doc/releases/squid.rst

```diff
@@ -0,0 +1,335 @@
+  options have been removed. Henceforth, users should use the new generic
+  option ``mon_cluster_log_level`` to control the cluster log level verbosity
+  for the cluster log file as well as for all external entities.
+
+RADOS
+~~~~~
+
+* RADOS: ``A POOL_APP_NOT_ENABLED`` health warning will now be reported if the
+  application is not enabled for the pool irrespective of whether the pool is
+  in use or not. Always tag a pool with an application using ``ceph osd pool
+  application enable`` command to avoid reporting of POOL_APP_NOT_ENABLED
+  health warning for that pool. The user might temporarily mute this warning
+  using ``ceph health mute POOL_APP_NOT_ENABLED``.
+* RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated due
+  to being prone to false negative results.  It's safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+* RADOS: For bug 62338 (https://tracker.ceph.com/issues/62338), we did not
+  choose to condition the fix on a server flag in order to simplify
+  backporting.  As a result, in rare cases it may be possible for a PG to flip
+  between two acting sets while an upgrade to a version with the fix is in
+  progress.  If you observe this behavior, you should be able to work around it
+  by completing the upgrade or by disabling async recovery by setting
+  osd_async_recovery_min_cost to a very large value on all OSDs until the
+  upgrade is complete: ``ceph config set osd osd_async_recovery_min_cost
+  1099511627776``
+* RADOS: A detailed version of the `balancer status` CLI command in the
+  balancer module is now available. Users may run `ceph balancer status detail`
+  to see more details about which PGs were updated in the balancer's last
+  optimization.  See https://docs.ceph.com/en/latest/rados/operations/balancer/
+  for more information.
+* RADOS: Read balancing may now be managed automatically via the balancer
+  manager module. Users may choose between two new modes: ``upmap-read``, which
+  offers upmap and read optimization simultaneously, or ``read``, which may be
+  used to only optimize reads. For more detailed information see
+  https://docs.ceph.com/en/latest/rados/operations/read-balancer/#online-optimization.
+* RADOS: BlueStore has been optimized for better performance in snapshot-intensive workloads.
+* RADOS: BlueStore RocksDB LZ4 compression is now enabled by default to improve average
+  performance and "fast device" space usage.
+* RADOS: A new CRUSH rule type, MSR (Multi-Step Retry), allows for more flexible EC
+  configurations.
+* RADOS: Scrub scheduling behavior has been improved.
+
+Crimson/Seastore
+~~~~~~~~~~~~~~~~
+
+* Crimson's first tech preview release!
+  Supporting RBD workloads on Replicated pools.
+  For more information please visit: https://ceph.io/en/news/crimson
+
+RBD
+~~~
+
+* RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
```

Comment:
is the '`' correct for the fast-diff?

Reply:
@ronen-fr Do you mean that you think it should be double-backticks instead of single-backticks?

Reply:
yes - as this is what's used some 5 lines below for try-netlink

Reply:
Fixed